Rubberduck.Core/UI/About/AboutControl.xaml

```diff
@@ -99,7 +106,7 @@
                     <Border Style="{StaticResource ReportButtonBorder}">
                         <TextBlock>
                             <Hyperlink Name="GitHubIssueUrl"
-                                       NavigateUri="https://github.com/rubberduck-vba/Rubberduck/issues/new"
+                                       NavigateUri="https://github.com/rubberduck-vba/Rubberduck/issues/new/choose"
```

Comment:
Loving this new link. We should probably add at least a "feature request" template on GH to use with.